tests/groups/test_db.py

```diff
@@ -43,24 +49,31 @@ async def test_multiple_documents(self, snapshot, fake2, mongo):
             base_processor(await mongo.users.find_one({"_id": user_1.id})),
         ]
 
-        assert (
-            await apply_transforms(
-                incomplete_users,
-                [AttachGroupsTransform(mongo), AttachPrimaryGroupTransform(mongo)],
-            )
-            == snapshot
+        complete_users = await apply_transforms(
+            incomplete_users,
+            [AttachGroupsTransform(mongo), AttachPrimaryGroupTransform(mongo)],
         )
 
+        for user in complete_users:
+            assert user == snapshot(
+                matcher=path_type(
+                    {"last_password_change": (datetime,), "password": (bytes,)}
+                )
+            )
```

Comment:
You can do one assertion by using regex path matching.

https://tophat.github.io/syrupy/#assertion-options

It would probably look like:
```py
path_type({
  ".*last_password_change": (datetime,),
  ".*password": (bytes,)
}, regex=True)
```